app/views/post/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_with url: post_index_path, model: Post do |form| %>
+  <%= form.label :title, 'Add a title for your post: ' %><br>
+  <%= form.text_field :title, placeholder: 'Title:' %> <br>
+  <%= form.label :text, 'Add text to your post:' %><br>
+  <%= form.text_area :text, placeholder: 'Text' %><br>
+  <%= form.submit "Create Post"%>
+<% end %>
```

Comment:
-  [Optional] Please make sure that the buttons in your code  matches the design 
![image](https://user-images.githubusercontent.com/74744787/216790501-1aef7fba-80b7-48ce-89a1-2e27072d7884.png)